docs/DeveloperGuide.md

```diff
@@ -1,34 +1,503 @@
+    * [4.4 Foreign Exchange Converter](#Foreign-Exchange-Converter-Shawn)
+    * [4.5 Import and Export Files](#45-Import-and-Export-Files-Wei-Shuang)
+    * [4.6 [Proposed] Colors](#46-Proposed-Colors)
+* [5.0 Appendix: Requirements](#50-Appendix-Requirements)
+    * [5.1 Product Scope](#51-Product-scope)
+        * [5.1.0 Target user profile](#510-Target-user-profile)
+        * [5.1.1 Value proposition](#511-Value-proposition)
+    * [5.2 User Stories](#52-User-Stories)
+    * [5.3 Use Cases](#53-Use-Cases)
+    * [5.4 Non-Functional Requirements](#54-Non-Functional-Requirements)
+    * [5.5 Glossary](#55-Glossary)
+* [6.0 Appendix: Instructions for Manual Testing](#60-Appendix-Instructions-for-Manual-Testing)
+    * [6.1 Launch and Shutdown](#61-Launch-and-Shutdown-Felix)
 
 
-## Product scope
-### Target user profile
+## 1.0 Introduction
 
-{Describe the target user profile}
+Trippie is a command-line app to plan any of your upcoming trips. It is tailored to match the needs of student travellers. Trippie lets users plan multiple trips through timetabling and also allows users to track their expenses overseas. Trippie is designed for users to enter their input quickly and efficiently.
 
-### Value proposition
+## 2.0 Setting up the project in your computer
+This sections gives an overview on how to set up the project in your computer.
 
-{Describe the value proposition: what problem does it solve?}
+**Prerequisites:**
+* JDK 11
+* IntelliJ IDE
 
-## User Stories
+<div markdown="span" class="alert alert-warning">
 
-|Version| As a ... | I want to ... | So that I can ...|
+⚠️ **Caution:**
+Follow the steps in the following guide precisely. Things will not work out if you deviate in some steps.
+</div>
+
+First, **fork** this repo, and **clone** the fork into your computer.
+
+If you plan to use Intellij IDEA (highly recommended):
+1. **Configure the JDK**: Follow the guide [_[se-edu/guides] IDEA: Configuring the JDK_](https://se-education.org/guides/tutorials/intellijJdk.html) to to ensure Intellij is configured to use **JDK 11**.
+1. **Import the project as a Gradle project**: Follow the guide [_[se-edu/guides] IDEA: Importing a Gradle project_](https://se-education.org/guides/tutorials/intellijImportGradleProject.html) to import the project into IDEA.
+:exclamation: Note: Importing a Gradle project is slightly different from importing a normal Java project.
+1. **Verify the setup**:
+   1. Run the `seedu.trippie.Trippie` and try a few commands.
+   2. [Run the tests](TestingGuide.md) to ensure they all pass. 
+
+## 3.0 Design
+
+This section explains the high-level design of the application. Given below is an overview of each component and a more detailed explanation of the architecture. Trippie is the main class which manages the initialization of the relevant classes and their execution.
+
+
+### 3.1 Architecture (Felix)
+
+<kbd>
+
+![UML Diagram Main](https://i.imgur.com/V3vUpoo.jpg)
+<center><i>Figure 1: Overall Architecture</i></center>
```

Comment:
![image](https://user-images.githubusercontent.com/49357313/97659338-b8f82f00-1aa9-11eb-8ff1-9465b40c60a0.png)
I think that the diagram has mixed elements of UML within it. For an architecture diagram, I believe it would be better to forgo the different dependency and multiplicity, while focusing more on the different component which interact with each other.